app/src/main/java/com/brainwallet/data/repository/LtcRepository.kt

```diff
@@ -55,15 +55,7 @@ interface LtcRepository {
         }
 
         override suspend fun fetchFeePerKb(): Fee {
-            return sharedPreferences.fetchWithCache(
```

Comment:
Oh...since the api server is serving a fresh object so that cache we always be stale @andhikayuana